apps/staking/src/api.ts

```diff
@@ -1,58 +1,37 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
@keyvankhademi especially take a look at this file as the requirements for the API have changed significantly and there are many new parts to account for all the new requirements.